### ISSUE TITLE
Upgrade to elm-pages 2.0

### ISSUE THREAD

**Issue by katjam** [CodeReadingClubs/www#39]
## Acceptance

- Good docs on how to add content
- better index & styling control

**Comment by katjam** [comment]
@Nikomus I'm part way through the port to elm-pages 2 and am removing the elm-ui stuff as part of it, so there may be some styling assist you can give soon. Maybe over the weekend?

**Comment by katjam** [comment]
Lol - one year on. I am actually doing this.

**Comment by Nikomus** [comment]
@katjam Will this help the site deploy on merge correctly? If not, we should probably wait to see what we need.

**Comment by katjam** [comment]
Yes, you are right. I am  just had a bit of time to lay a bit more groundwork - but I will get the deploy working today as a separate short task
